### ISSUE TITLE
Wrong translation title

### ISSUE THREAD

**Issue by JasonCard** [dotnet/AspNetCore.Docs#19462]
Transferring [https://github.com/dotnet/AspNetCore.Docs.zh-tw/issues/147](https://github.com/dotnet/AspNetCore.Docs.zh-tw/issues/147) from @u8989332

English: Use social sign-in provider authentication without ASP.NET Core Identity

Current translation: 使用不 ASP.NET Core 的社交登入提供者驗證Identity

Revised translation: 使用沒有ASP.NET Core Identity的社交登入提供者驗證

文件詳細資料
⚠ 請勿編輯此區段。 這是 docs.microsoft.com ➟ GitHub 問題連結的必要項。

ID: 6f9d7ad5-47d6-80bf-88fe-fac3a74151a3
Version Independent ID: e64816f4-3391-5aab-31c2-ab4f915bc7f6
Content: 不 ASP.NET Core 的 Facebook、Google 及外部提供者驗證Identity
Content Source: aspnetcore/security/authentication/social/social-without-identity.md
Product: aspnet-core
Technology: aspnetcore-security
GitHub Login: @Rick-Anderson
Microsoft Alias: riande

**Comment by JasonCard** [comment]
Rick-Anderson commented 13 hours ago
cc @guardrex @scottaddie
@JasonCard please transfer this issue to https://github.com/dotnet/AspNetCore.Docs/issues and I'll add Identity to the no loc list. It's always uppercase.


Rick-Anderson commented 13 hours ago
Actually it looks like it's not translating Identity and that's what we want.
@u8989332 Identity is a product and shouldn't be translated. Why do you think it should be translated?


@Rick-Anderson The subject ASP.NET Core Identity is a major library so it should be bundled together, not separate.


Rick-Anderson commented 10 hours ago
@u8989332 I see now

Current translation:: Use a social login provider that is not ASP.NET Core to verify Identity

Revised translation: Use social login provider authentication without ASP.NET Core Identity

Maybe we need no loc: ASP.NET Core Identity,


@Rick-Anderson Yes, the English current translation means I mention the semantic problem with Traditional-Chinese.

#146 is the same problem.

**Comment by Rick-Anderson** [comment]
@guardrex @JasonCard can I just change
`no-loc: [`
to
`no-loc: [ASP.NET Core Identity, `  <-- ends with space

**Comment by guardrex** [comment]
Put quotes around it ...

```
no-loc: ["ASP.NET Core Identity", 
```

... with the trailing space after the comma.

**Comment by JasonCard** [comment]
I think we should not use double quotes from looking at the examples [here](https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=master#non-localized-strings):
```
no-loc: [Global, Strings, to be, Ignored]
```
FYI, in the past, the no-loc feature did not support the terms connected by period like "ASP.NET", but that was fixed, so I believe this should work now:
```
no-loc: [ASP.NET Core Identity,  
```

**Comment by guardrex** [comment]
Oh ... I thought we worked out doubles previously. Our current string is ...

```
no-loc: [cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
```

... so, we'll go to ....

```
no-loc: [ASP.NET Core Identity, cookie, Cookie, Blazor, Blazor Server, Blazor WebAssembly, Identity, "Let's Encrypt", Razor, SignalR]
```

... with only doubles on the single-quoted "Let's Encrypt"? Is that correct?

**Comment by JasonCard** [comment]
Oh, sorry, I didn't realize that you're already using the double quotes. I confirmed that it's working correctly with double quotes, so it's okay to use the double quotes!

**Comment by Rick-Anderson** [comment]
Fixed in #19578
